examples/heat_demand_example.py

```diff
@@ -28,7 +28,7 @@
 import demandlib.bdew as bdew
 
 # read example temperature series
-datapath = os.path.join(os.path.dirname(__file__), "example_data.csv")
+datapath = os.path.join(os.getcwd(), "example_data.csv")
```

Comment:
This will not work if people try to "python ./examples/heat_demand_example.py". But this is probably acceptable.